packages/react-liveness/src/components/FaceLivenessDetector/shared/Hint.tsx

```diff
@@ -87,18 +95,28 @@ export const Hint: React.FC<HintProps> = ({ hintDisplayText }) => {
       if (isCheckFaceDetectedBeforeStart) {
         if (faceMatchStateBeforeStart === FaceMatchState.TOO_MANY) {
           return (
-            <Toast>{FaceMatchStateStringMap[faceMatchStateBeforeStart]}</Toast>
+            <Toast size="large" variation="primary">
+              {FaceMatchStateStringMap[faceMatchStateBeforeStart]}
+            </Toast>
           );
         }
-        return <Toast>{hintDisplayText.hintMoveFaceFrontOfCameraText}</Toast>;
+        return (
+          <Toast size="large" variation="primary">
+            {hintDisplayText.hintMoveFaceFrontOfCameraText}
+          </Toast>
+        );
       }
 
       // Specifically checking for false here because initially the value is undefined and we do not want to show the instruction
       if (
         isCheckFaceDistanceBeforeRecording &&
         isFaceFarEnoughBeforeRecordingState === false
       ) {
-        return <Toast>{hintDisplayText.hintTooCloseText}</Toast>;
+        return (
+          <Toast size="large" variation="primary">
```

Comment:
nonblocking nit: Might be nice to declare each of these Toast components as named Components outside the function since they don't  seem to change. This one could be called `TooCloseHint` or something.

Reply:
Will do in a follow up, i'll add a note to myself